ui/components-react/src/components-react/filter-builder/FilterBuilder.tsx

```diff
@@ -95,10 +94,16 @@ export interface PropertyFilterBuilderRendererProps {
   ) => React.ReactNode;
   /** Custom renderer for property selector in rule. */
   propertyRenderer?: (name: string) => React.ReactNode;
-  /** Specifies how deep rule groups can be nested. */
+  /** Specifies how deep rule groups can be nested.
+   * @deprecated in 4.8.0. Nesting is no longer supported moving forward.
+   */
   ruleGroupDepthLimit?: number;
   /** Specifies whether component is disabled or not. */
   isDisabled?: boolean;
+  /** Size to render the component. If undefined, defaults to iTwinUI "small" size. */
+  size?: "medium" | "large";
+  /** Controls whether the group operator is toggle-able. */
+  isOperatorToggleDisabled?: boolean;
```

Comment:
I would suggest renaming to `isGroupOperatorDisabled` or `isLogicalOperatorDisabled`.